### PR TITLE
Create User Calendar & Save Preferences

### LINKED ISSUE #5: User's 'Welcome' page
**Describe the functionality or user interaction with the app**
As a user, when I visit '/', I am on the 'Welcome' page and see a brief description of the app
**Describe associated tests**
Test 'Welcome' page exists and contains a description of app
**Related Issues**

**Tasks**
Create Structure
- [ ] Set up landing component structure
- [ ] add styling
- [ ] include description
- [ ] create snapshot testing
- [ ] create Pack?

Screen Shot 2020-04-04 at 7 22 46 PM

### LINKED ISSUE #8: User's Calendar
**Describe the functionality or user interaction with the app**
As a User, when I visit my 'profile' page, I see my calendar where I can update my schedule.
**Describe associated tests**
Test calendar is rendered on profile page. Test user can update it.
**Related Issues**

**Tasks**
Create Structure
- [ ] Set up landing component structure
- [ ] Incorporate treat map
- [ ] add styling
- [ ] create snapshot testing
- [ ] create Pack?

Screen Shot 2020-04-04 at 7 18 20 PM

### PR DESCRIPTION
**Pull request by joel-oe-lacey**
### Sidebar Checklist
- [X] Request reviewers
- [X] Assign yourself and other contributors
- [X] Link to project

### Issues Resolved
Resolves #8, Resolves #5 

### Problem Addressed
Previously, a user was unable to save their activity preferences, and unable to see events that they had upcoming on their calendar.

### Solution Implemented
This change looks to resolve that by creating a calendar display for the week's tasks. It also adds functionality to save up to three activity preferences and post those. In addition it updates some styling and component structuring for a cleaner UI.

### Other Notes
None.

### CODE REVIEW

File: src/components/Calendar/Calendar.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    header: {
+        fontSize: 27,
+        fontWeight: 'bold'
+    },
+    day: {
+        fontSize: 25,
+        marginTop: 30
+    },
+    categories: {
+        backgroundColor: "#fff",
```
Comment (wrenisit) [inline src/components/Calendar/Calendar.js:86]: Will this background color be different than the ones on the other pages?

File: src/components/Calendar/TreatEvent.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            <Text style={styles.header}>{title}</Text>
+            <Text style={styles.header}>{`${duration} Mins`}</Text>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    container: {
+        // flex: 1,
+        backgroundColor: '#fff',
```
Comment (wrenisit) [inline src/components/Calendar/TreatEvent.js:18]: Same background color question here.

File: src/components/Profile/Profile.js [hunk 3]
```diff
@@ -1,68 +1,46 @@
...
-    //use that check to remove from selected array if true and clicked
-
-    if (data) {
-        catDisplay = <FlatList
-            data={data.getCategories}
-            renderItem={({ item, index }) => (
-                <SuggestedTreat key={index} title={item.name} selectCategory={selectCategory} />
-            )}
-        />
+        fetch('https://treat-yo-self-bjtw.herokuapp.com/api/v1/suggestions ', options)
```
Comment (wrenisit) [inline src/components/Profile/Profile.js:23]: This looks great. Thank you for using this endpoint.

File: src/components/Calendar/Calendar.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, SectionList } from 'react-native';
+import TreatEvent from './TreatEvent';
+// import { gql } from 'apollo-boost';
+// import { useQuery } from '@apollo/react-hooks';
+import { mockEvents } from '../../mockData/mockTreat';
+
+export default function Calendar() {
```
Comment (StewartTFrancis) [inline src/components/Calendar/Calendar.js:8]: Good using mocks.

File: src/components/Calendar/Calendar.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    //do calcs here to group into a single date object 
+    //dont need to parse float just JSON parse on fetch 
+
+    const calendarEvents = mockEvents.reduce((eventByDay, event) => {
+        let {eventName, eventStartTime, eventEndTime} = event;
+        eventStartTime = new Date(parseFloat(eventStartTime));
+        eventEndTime = new Date(parseFloat(eventEndTime));
+        const duration = Math.floor((eventEndTime.getTime() - eventStartTime.getTime()) / 60000);
```
Comment (StewartTFrancis) [inline src/components/Calendar/Calendar.js:16]: I'd recommend something like moment.js to avoid any potential timezone/hard coded constant/etc issues.

Comment (wrenisit) [reply src/components/Calendar/Calendar.js:16]: Thank you for this suggestion. I will look into implementing this in a future branch.

**Pull request merged**